### PR TITLE
A new face for the Sultan of ML deployment

### PR DESCRIPTION
**Pull request by sheikhartin**
### Description

The pull request title may seem strange, I just wanted to add some passion! 👑😅 Gradio is a cool and professional project, but it doesn't have a good identity (I mean README)... I would like to help to improve the appearance of Gradio itself in addition to the beauty of its outputs!

A quick overview of the changes I made:

- `CITATION.cff`: To display `Cite this repository` in the project sidebar and help to find the paper.
- `readme_template.md`: Update and add some shields, and adding frequently used links to the header section to avoid more repetition. (I like this part the most :))
- `quickstart.md`: Typo fix and adding more explanations (including updating some links with duplicate texts).
- `render_readme.py`: Using a new technique for rendering, making the titles one step smaller, and removing an extra line in the generated code block. Why should the `quickstart.md` file titles be smaller? Each section is a subset of its parent, so when `quickstart.md` wants to add it to the `README.md` file, it should come as a subset, not the main title with `h1`/`#`.
 
Closes: #1438

### Checklist

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

I would like to see some good changes in version 3.2! (Due by August 12, 2022)

Keep going guys

### CODE REVIEW

File: guides/1)getting_started/1)quickstart.md [hunk 1]
```diff
@@ -1,110 +1,108 @@
...
 
-Gradio allows you to **build demos and share them, all in Python.** And usually in just a few lines of code! So let's get started. 
+Gradio allows you to **build demos and share them, all in Python.** And usually in just a few lines of code! So let's get started.
 
 ## Hello, World
 
 To get Gradio running with a simple "Hello, World" example, follow these three steps:
 
-<span>1.</span> Install Gradio from pip. Note, the minimal supported Python version is 3.7.
+1\. Install Gradio using PIP:
```
Comment (abidlabs) [inline guides/1)getting_started/1)quickstart.md:15]:
```suggestion
1\. Install Gradio from pip:
```

File: readme_template.md [hunk 2]
```diff
@@ -1,25 +1,21 @@
...
-    <img src="https://img.shields.io/twitter/follow/gradio.svg?style=social&label=Follow" alt="twitter">
-</a>
-</p>
+<div align="center">
+
+  [<img src="readme_files/gradio.svg" alt="gradio" width=300>](https://gradio.app)<br><br>
+  <em>Build & share delightful machine learning apps easily</em>
+
+  [![CircleCI](https://img.shields.io/circleci/build/gh/gradio-app/gradio)](https://circleci.com/gh/gradio-app/gradio)
+  [![Codecov branch](https://img.shields.io/codecov/c/github/gradio-app/gradio/main)]()
```
Comment (abidlabs) [inline readme_template.md:7]: I prefer the previous codecov badge. But all the other ones look great!

File: guides/1)getting_started/1)quickstart.md [hunk 3]
```diff
@@ -1,110 +1,108 @@
...
 
 ```bash
 pip install gradio
 ```
 
-<span>2.</span> Run the code below as a Python script or in a Python notebook (or in a  [colab notebook](https://colab.research.google.com/drive/18ODkJvyxHutTN0P5APWyGFO_xwNcgHDZ?usp=sharing)).
+2\. Run the code below as a Python script or in a Jupyter Notebook ([Google Colab](https://colab.research.google.com/drive/18ODkJvyxHutTN0P5APWyGFO_xwNcgHDZ?usp=sharing) also is a good option):
```
Comment (abidlabs) [inline guides/1)getting_started/1)quickstart.md:21]:
```suggestion
2\. Run the code below as a Python script or in a Jupyter Notebook (or [Google Colab](https://colab.research.google.com/drive/18ODkJvyxHutTN0P5APWyGFO_xwNcgHDZ?usp=sharing)):
```

File: guides/1)getting_started/1)quickstart.md [hunk 4]
```diff
@@ -1,110 +1,108 @@
...
 
-You'll notice that in order to make the demo, we created a `gradio.Interface`. This `Interface` class can wrap any Python function with a user interface. In the example above, we saw a simple text-based function. But the function could be anything from music generator to a tax calculator to the prediction function of a pretrained machine learning model.
+You'll notice that in order to make the demo, we created a `gradio.Interface`. This `Interface` class can wrap any Python function with a user interface. In the example above, we saw a simple text-based function, but the function could be anything from music generator to a tax calculator to the prediction function of a pretrained machine learning model.
 
 The core `Interface` class is initialized with three required parameters:
 
--   `fn`: the function to wrap a UI around
--   `inputs`: which component(s) to use for the input, e.g. `"text"` or `"image"` or `"audio"` 
--   `outputs`: which component(s) to use for the output, e.g. `"text"` or `"image"` `"label"`
+- `fn`: The function to wrap a UI around.
```
Comment (abidlabs) [inline guides/1)getting_started/1)quickstart.md:35]: Let's stick to lower case and without the period since these are not full sentences

**Comment by abidlabs** [comment]
Wow thanks for the contribution @sheikhartin! Love the changes (and the passion 😄). I made a few changes to the PR, but overall, I think this is a great improvement to the readme and quickstart.

cc @aliabid94 you may have some thoughts as well.

**Comment by sheikhartin** [comment]
Thank you too @abidlabs, for respecting the enthusiasm of the contributors and not disappointing them to continue contributions! 🥲

After about 4 years of working with Python, I still don't know if I should write `pip` or `PIP`! But I usually use the second one because it is an abbreviation and capital letters are more common... (`CIA` or `cia` is a good case for discussion)

I understand and agree with you about the CircleCI sheilds! :)

**Comment by abidlabs** [comment]
This looks perfect, thank you again @sheikhartin! I'll go ahead and merge it in tomorrow unless @aliabid94 or @aliabd have any additional feedback.

**Comment by abidlabs** [comment]
Also notices that this fixes #1438 so editing the PR description to include that

**Review by abidlabs — APPROVED** [review]

**Comment by sheikhartin** [comment]
These two codes contains some white space and this is the reason that, apart from replacing single quotes with double quotes, these items are also displayed.

Sepia filter:
https://github.com/gradio-app/gradio/blob/74a2110a705b74bbff32c588b120e71d781d7c1d/demo/sepia_filter/run.py#L6

Blocks flipper:
https://github.com/gradio-app/gradio/blob/74a2110a705b74bbff32c588b120e71d781d7c1d/demo/blocks_flipper/run.py#L22

**Comment by abidlabs** [comment]
Looks good. Merging!

**Pull request merged**